docs/index.md

```diff
@@ -1,4 +1,128 @@
-<img style="float: left; padding-top: 7px; padding-right: 10px; padding-bottom: 20px;" src="images/atomist-logo-source.jpg">
+<img style="float:left; padding-top:7px; padding-right:10px; padding-bottom:15px;" src="images/atomist-logo.png">
+
+# Atomist Overview
+
+Atomist enables you to ship software in less time and respond to
+operational events more quickly.  By automating tasks, integrating
+tools, and putting timely, correlated, and actionable information in
+front of your team, Atomist makes software development and operational
```

Comment:
"in front of your team" and the team phrasing in this section feels like we could simplify it. e.g. "By automating tasks, integrating tools, and ~putting~ *providing* timely, correlated, and actionable information ~in front of your team~, Atomist makes software ~development and operational response~ *delivery* faster, more standardized, and more fun.